collections/editor/includes/queryform.php

```diff
@@ -85,15 +85,15 @@
 			if(!$crowdSourceMode){
 				?>
 				<div class="fieldGroupDiv">
-					<div title="<?php echo $LANG['WILD_EXPLAIN']; ?>">
+					<div class="fieldDiv" title="<?php echo $LANG['WILD_EXPLAIN']; ?>">
```

Comment:
I returned some original styling here as well. And for a several other <div> elements in this same file.